cloud/aws/deploytf/api.go

```diff
@@ -99,17 +96,26 @@ func (n *NitricAwsTerraformProvider) Api(stack cdktf.TerraformStack, name string
 	}
 
 	nitricServiceTargets := map[string]service.Service{}
-	for _, p := range openapiDoc.Paths {
-		for _, op := range p.Operations() {
-			if v, ok := op.Extensions["x-nitric-target"]; ok {
-				if targetMap, isMap := v.(map[string]any); isMap {
+	for _, apiPath := range openapiDoc.Paths {
+		for _, pathOperation := range apiPath.Operations() {
+			if apiNitricTarget, ok := pathOperation.Extensions["x-nitric-target"]; ok {
+				if targetMap, isMap := apiNitricTarget.(map[string]any); isMap {
 					serviceName := targetMap["name"].(string)
```

Comment:
```suggestion
					serviceName, ok := targetMap["name"].(string)
					if !ok {
					    return fmt.Errorf("missing or invalid 'name' field in x-nitric-target for path %s on API 				%s", pathOperation.OperationID, name)
					}
```

Reply:
Probably not reachable with the typical Nitric flow as well, but doesn't hurt :)